samples/sample_encode/include/pipeline_encode.h

```diff
@@ -401,7 +428,12 @@ class CEncodingPipeline
     virtual mfxStatus AllocateSufficientBuffer(mfxBitstreamWrapper& bs);
     virtual mfxStatus FillBuffers();
     virtual mfxStatus LoadNextFrame(mfxFrameSurface1* pSurf);
-    virtual void LoadNextControl(mfxEncodeCtrlWrap*& pCtrl, mfxU32 encSurfIdx);
+    virtual void LoadNextControl(mfxEncodeCtrl*& pCtrl, mfxU32 encSurfIdx);
```

Comment:
Added new methods... Where is implementation?

Reply:
@alariono please see pipeline_encode.cpp:2696
It`s just signature change.